pymodbus/datastore/context.py

```diff
@@ -27,6 +31,43 @@ def decode(self, fx):
         """
         return self._fx_mapper[fx]
 
+    async def async_getValues(self, fc_as_hex: int, address: int, count: int = 1) -> list[int | bool] | ExceptionResponse:
```

Comment:
ExceptionResponse is not a valid return from getValues, however it can return None in the list.

Reply:
Same for here, it will return Response https://github.com/pymodbus-dev/pymodbus/blob/dev/pymodbus/datastore/remote.py#L129 if it is error one, even message execute assumes so, https://github.com/pymodbus-dev/pymodbus/blob/dev/pymodbus/bit_write_message.py#L97 . I'll add the None to the list